### PR TITLE
IGNITE-14129 Add documentation for .NET thin client service invocation

### PR DESCRIPTION
**Pull request by ptupitsyn**

### CODE REVIEW

**Review by Nikita-tech-writer — APPROVED** [review]

**Comment by Nikita-tech-writer** [comment]
@dmagda @Mmuzaf Could you please merge this PR and cherry pick the changes to the 2.10?

File: docs/_docs/thin-clients/dotnet-thin-client.adoc [hunk 1]
```diff
@@ -235,6 +235,20 @@ task named `org.foo.bar.AddOneTask` passing `1` as an input parameter:
 include::{sourceCodeFile}[tag=client-compute-task,indent=0]
 -------------------------------------------------------------------------------
 
+== Executing Ignite Services
+
+You can use the `IServicesClient` API to invoke an link:services/services[Ignite Service] that
+is *already deployed* in the cluster.
```
Comment (dmagda) [inline docs/_docs/thin-clients/dotnet-thin-client.adoc:241]: Can we cross-link to a page/section explaining how the services can be deployed? Developers that start with thin clients might not be aware of that.

File: docs/_docs/thin-clients/dotnet-thin-client.adoc [hunk 2]
```diff
@@ -235,6 +235,20 @@ task named `org.foo.bar.AddOneTask` passing `1` as an input parameter:
...
+
+The example below shows how to invoke the service named `MyService`:
+[source, csharp]
+-------------------------------------------------------------------------------
+include::{sourceCodeFile}[tag=client-services,indent=0]
+-------------------------------------------------------------------------------
+
+The deployed service can be implemented using Java or .NET.
```
Comment (dmagda) [inline docs/_docs/thin-clients/dotnet-thin-client.adoc:249]: I would say "The thin client allows to execute services implemented in Java or C#"

**Comment by dmagda** [comment]
@Nikita-tech-writer, please consider the minor suggestions and work with Maxim to merge the changes to the master and 2.10.

File: docs/_docs/thin-clients/dotnet-thin-client.adoc (hunk 1 above)
Comment (ptupitsyn) [reply docs/_docs/thin-clients/dotnet-thin-client.adoc]: Done

File: docs/_docs/thin-clients/dotnet-thin-client.adoc (hunk 2 above)
Comment (ptupitsyn) [reply docs/_docs/thin-clients/dotnet-thin-client.adoc]: Fixed

**Comment by Nikita-tech-writer** [comment]
@Mmuzaf, I believe that now we can merge the changes to the master and to the Ignite 2.10.

**Review by Nikita-tech-writer — APPROVED** [review]

**Review by Mmuzaf — APPROVED** [review]

**Pull request merged**